src/App.js

```diff
@@ -13,7 +13,7 @@ function App() {
     <Router>
       <Switch>
         <Route exact path="/" component={Login}/>
-        <Route exact path="/welcome/:id" component={Welcome}/>
+        <Route path= "/welcome/newUser" component={Welcome}/>
```

Comment:
Please, consider replace this change, instead, try this: `<Route exact path="/welcome/:id/" component={Welcome}/>` CC @tanyalej .

Reply:
Mock data doesn't work with your proposal change `<Route path= "/welcome/newUser" component={Welcome}/>` CC @tanyalej